qiskit/chemistry/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_uvccsd_factory.py

```diff
@@ -0,0 +1,145 @@
+                an appropriate expectation based on the operator and backend. When using Aer
+                qasm_simulator backend, with paulis, it is however much faster to leverage custom
+                Aer function for the computation but, although VQE performs much faster
+                with it, the outcome is ideal, with no shot noise, like using a state vector
+                simulator. If you are just looking for the quickest performance when choosing Aer
+                qasm_simulator and the lack of shot noise is not an issue then set `include_custom`
+                parameter here to ``True`` (defaults to ``False``).
+            include_custom: When `expectation` parameter here is None setting this to ``True`` will
+                allow the factory to include the custom Aer pauli expectation.
+        """
+        self._quantum_instance = quantum_instance
+        self._optimizer = optimizer
+        self._initial_point = initial_point
+        self._expectation = expectation
+        self._include_custom = include_custom
+
+    @property
+    def quantum_instance(self) -> QuantumInstance:
+        """Getter of the quantum instance."""
+        return self._quantum_instance
+
+    @quantum_instance.setter
+    def quantum_instance(self, q_instance: QuantumInstance) -> None:
+        """Setter of the quantum instance."""
+        self._quantum_instance = q_instance
+
+    @property
+    def optimizer(self) -> Optimizer:
+        """Getter of the optimizer."""
+        return self._optimizer
+
+    @optimizer.setter
+    def optimizer(self, optimizer: Optimizer) -> None:
+        """Setter of the optimizer."""
+        self._optimizer = optimizer
+
+    @property
+    def initial_point(self) -> np.ndarray:
+        """Getter of the initial point."""
+        return self._initial_point
+
+    @initial_point.setter
+    def initial_point(self, initial_point: np.ndarray) -> None:
+        """Setter of the initial point."""
+        self._initial_point = initial_point
+
+    @property
+    def expectation(self) -> ExpectationBase:
+        """Getter of the expectation."""
+        return self._expectation
+
+    @expectation.setter
+    def expectation(self, expectation: ExpectationBase) -> None:
+        """Setter of the expectation."""
+        self._expectation = expectation
+
+    @property
+    def include_custom(self) -> bool:
+        """Getter of the ``include_custom`` setting for the ``expectation`` setting."""
+        return self._include_custom
+
+    @include_custom.setter
+    def include_custom(self, include_custom: bool) -> None:
+        """Setter of the ``include_custom`` setting for the ``expectation`` setting."""
+        self._include_custom = include_custom
+
+    def get_solver(self, transformation: BosonicTransformation) -> MinimumEigensolver:
+        """Returns a VQE with a UVCCSD wavefunction ansatz, based on ``transformation``.
```

Comment:
UVCC

Reply:
I'd rather keep it like this

Reply:
same as above. Resolving this.

Reply:
Ok, I think its rather confusing given the var form is UVCC - but fine.